app/views/commission_history_mailer/commission_change_info.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Notification about commission change</h1>
+
+<p>
+  Validator <%= link_to @validator_title, validator_url(@validator) %> changed it's commission.
```

Comment:
This URL is not working, because it uses ID as a param by default and we use account in our routes. Also network is missing.